hack/quickstart-setup.sh

```diff
@@ -0,0 +1,110 @@
+  KUADRANT_ISTIO_KUSTOMIZATION=${KUADRANT_ISTIO_KUSTOMIZATION}?ref=${KUADRANT_BRANCH}
+  KUADRANT_CERT_MANAGER_KUSTOMIZATION=${KUADRANT_CERT_MANAGER_KUSTOMIZATION}?ref=${KUADRANT_BRANCH}
+  KUADRANT_METALLB_KUSTOMIZATION=${KUADRANT_METALLB_KUSTOMIZATION}?ref=${KUADRANT_BRANCH}
+fi
+
+# Make temporary directory
+mkdir -p ${TMP_DIR}
+
+KUADRANT_CLUSTER_NAME=kuadrant-local
+KUADRANT_NAMESPACE=kuadrant-system
+
+echo "Do you want to set up a DNS provider? (y/N)"
+read SETUP_PROVIDER </dev/tty
+if [[ "$SETUP_PROVIDER" =~ ^[yY]$ ]]; then
+  requiredENV
+fi
+
+# Kind delete cluster
+${KIND_BIN} delete cluster --name ${KUADRANT_CLUSTER_NAME}
+
+# Kind create cluster
+${KIND_BIN} create cluster --name ${KUADRANT_CLUSTER_NAME} --config=- <<< "$(curl -s ${KUADRANT_REPO_RAW}/utils/kind-cluster.yaml)"
+kubectl config use-context kind-${KUADRANT_CLUSTER_NAME}
+
+# Create namespace
+kubectl create namespace ${KUADRANT_NAMESPACE}
+
+# Install gateway api
+echo "Installing Gateway API in ${KUADRANT_CLUSTER_NAME}"
+${KUSTOMIZE_BIN} build ${KUADRANT_GATEWAY_API_KUSTOMIZATION} | kubectl apply -f -
+
+# Install istio
+echo "Installing Istio in ${KUADRANT_CLUSTER_NAME}"
+${KUSTOMIZE_BIN} build ${KUADRANT_ISTIO_KUSTOMIZATION} | kubectl apply -f -
+kubectl -n istio-system wait --for=condition=Available deployment istio-operator --timeout=300s
+kubectl apply -f ${KUADRANT_REPO_RAW}/config/dependencies/istio/sail/istio.yaml
+
+# Install cert-manager
+echo "Installing cert-manager in ${KUADRANT_CLUSTER_NAME}"
+${KUSTOMIZE_BIN} build ${KUADRANT_CERT_MANAGER_KUSTOMIZATION} | kubectl apply -f -
+echo "Waiting for cert-manager deployments to be ready"
+kubectl -n cert-manager wait --for=condition=Available deployments --all --timeout=300s
+
+# Install metallb
+echo "Installing metallb in ${KUADRANT_CLUSTER_NAME}"
+${KUSTOMIZE_BIN} build ${KUADRANT_METALLB_KUSTOMIZATION} | kubectl apply -f -
+echo "Waiting for metallb-system deployments to be ready"
+kubectl -n metallb-system wait --for=condition=Available deployments controller --timeout=300s
+kubectl -n metallb-system wait --for=condition=ready pod --selector=app=metallb --timeout=60s
+kubectl apply -n metallb-system -f - <<< "$(curl -s ${KUADRANT_REPO_RAW}/utils/docker-network-ipaddresspool.sh | bash -s -- kind)"
+
+# Install kuadrant
+echo "Installing Kuadrant in ${KUADRANT_CLUSTER_NAME}"
+${KUSTOMIZE_BIN} build ${KUADRANT_DEPLOY_KUSTOMIZATION} | kubectl apply -f -
+kubectl -n ${KUADRANT_NAMESPACE} patch deployment kuadrant-operator-controller-manager --type='merge' -p '{"spec":{"template":{"spec":{"containers":[{"name":"manager","image":"'"${KUADRANT_IMAGE}"'"}]}}}}'
+
+# Configure managedzone
+if [ ! -z "$DNS_PROVIDER" ]; then
+  configureController ${KUADRANT_CLUSTER_NAME} ${KUADRANT_NAMESPACE}
+fi
+
+# Deploy kuadrant
+kubectl -n ${KUADRANT_NAMESPACE} apply -f ${KUADRANT_REPO_RAW}/config/samples/kuadrant_v1beta1_kuadrant.yaml
```

Comment:
I think it might be good to echo out a link to the follow on walkthrough WDYT?

Reply:
100% agree

Reply:
I guess this should be the link to the location on kuadrant.io rather than github once we have the doc PR merged?

Reply:
Ah nevermind, I see the link is already there - updated now